providers/implementations/signature/ecdsa_sig.c

```diff
@@ -352,8 +500,14 @@ static int ecdsa_digest_signverify_init(void *vctx, const char *mdname,
 #ifdef FIPS_MODULE
     ctx->verify_message = 1;
 #endif
-    if (!ecdsa_signverify_init(ctx, ec, params, operation, desc)
-        || !ecdsa_setup_md(ctx, mdname, NULL, desc))
+    if (!ecdsa_signverify_init(vctx, ec, ecdsa_set_ctx_params, params,
+                               operation, desc))
+        return 0;
+
+    if (mdname != NULL
+        /* was dsa_setup_md already called in dsa_signverify_init()? */
```

Comment:
s/dsa_setup_md/ecdsa_setup_md/

Reply:
Okie

Reply:
This wasn't fixed?

Reply:
...  I "fixed" it in dsa_sig.c.  Now fixing again